src/_static/js/launchcode.js

```diff
@@ -42,9 +42,26 @@ function insertAdmonitionIndicators() {
 
 }
 
+function addTargetToExternalLinks() {
+
+  const links = document.getElementsByTagName("a");
+
+  for (let i = 0; i < links.length; i++) {
+    const link = links[i];
+    const url = link["href"];
+    if (!(url.includes("launchcode.org") || url.includes("localhost"))) {
```

Comment:
Can you add `file://`? I don't serve my local site using a web server, I load it directly from the file system.

Reply:
Sure thing.